docs/use cases/README.md

```diff
@@ -63,6 +63,36 @@
 
 ### Діаграма прецедентів для учасника проєкту
 
+<div style="
+    border-radius:4px;
+    border: 1px solid #cfd7e6;
+    padding: 1em;">
+
+@startuml
+
+    actor "Учасник проєкту" as ProjectMember #e8ffe9
+
+    usecase "Управління\nзавданнями" as TaskManagement #e4ffdb
+
+    usecase "Створити\nзавдання" as CreateTask 
+    usecase "Редагувати\nзавдання" as EditTask 
+    usecase "Видалити\nзавдання" as DeleteTask 
+    usecase "Коментувати\nзавдання" as CommentTask 
+    usecase "Вийти\nз проєкту" as LogOut #e4ffdb
+
+
+    ProjectMember -u-> LogOut
+    ProjectMember -d-> TaskManagement
+
+    TaskManagement <.l. CreateTask
+    TaskManagement <.d. EditTask
+    TaskManagement <.d. CommentTask
+    TaskManagement <.r. DeleteTask
```

Comment:
scheme should look like this:
![Screenshot_20241015_220826](https://github.com/user-attachments/assets/d5aa6c10-0428-4f20-bbd2-f3804006639f)